demux/cli/indexreport.py

```diff
@@ -0,0 +1,63 @@
+import click
+import logging
+
+from demux.utils.indexreport import IndexReport
+from demux.constants import INDEX_REPORT_HEADER
+
+log = logging.getLogger(__name__)
+
+
+@click.group()
+def indexreport():
+    """Index report commands"""
+    pass
+
+
+@indexreport.command()
+@click.option(
+    "-i",
+    "--index-report-path",
+    type=click.File(mode="r"),
+    required=True,
+    help="Relative path to bcl2fastq indexcheck report (laneBarcode.html)",
+)
+@click.option(
+    "-o",
+    "--out-dir",
+    type=click.Path(exists=True),
+    required=True,
+    help="Relative path to the output directory",
```

Comment:
Meant to be a pointer to the directory where you want the summary. It could be an absolute path here, but the code works on both absolute and relative ones. I'll change it to absolute to avoid ambiguity.